documentation/content/en/books/arch-handbook/boot/_index.adoc

```diff
@@ -469,43 +472,43 @@ With the MBR loaded to memory, the actual loop for searching the FreeBSD slice b
 ....
 	mov $0x1,%cx		 # Two passes
 main.1:
-	mov $0x8dbe,%si # Partition table
+	mov $MEM_BUF+PRT_OFF,%si # Partition table
 	movb $0x1,%dh		 # Partition
 main.2:
-	cmpb $0xa5,0x4(%si)	 # Our partition type?
+	cmpb $PRT_BSD,0x4(%si)	 # Our partition type?
 	jne main.3		 # No
 	jcxz main.5		 # If second pass
 	testb $0x80,(%si)	 # Active?
 	jnz main.5		 # Yes
 main.3:
 	add $0x10,%si		 # Next entry
 	incb %dh		 # Partition
-	cmpb $0x5,%dh		 # In table?
+	cmpb $0x1+PRT_NUM,%dh		 # In table?
 	jb main.2		 # Yes
 	dec %cx			 # Do two
 	jcxz main.1		 #  passes
 ....
 
-.[.filename]#sys/boot/i386/boot2/boot1.S# [[boot-boot1-find-part]]
+.[.filename]#stand/i386/boot2/boot1.S# [[boot-boot1-find-part]]
 If a FreeBSD slice is identified, execution continues at `main.5`. Note that when a FreeBSD slice is found `%si` points to the appropriate entry in the partition table, and `%dh` holds the partition number. We assume that a FreeBSD slice is found, so we continue execution at `main.5`:
 
 [.programlisting]
 ....
 main.5:
-	mov %dx,0x900			   # Save args
-	movb $0x10,%dh			   # Sector count
+	mov %dx,MEM_ARG			   # Save args
+	movb $NSECT,%dh			   # Sector count
 	callw nread			   # Read disk
-	mov $0x9000,%bx			   # BTX
+	mov $MEM_BTX,%bx		   # BTX
 	mov 0xa(%bx),%si		   # Get BTX length and set
 	add %bx,%si			   #  %si to start of boot2.bin
-	mov $0xc000,%di			   # Client page 2
-	mov $0xa200,%cx			   # Byte
+	mov $MEM_USR+SIZ_PAG*2,%di	   # Client page 2
+	mov $MEM_BTX+(NSECT-1)*SIZ_SEC,%cx # Byte
```

Comment:
There are also some tabs here that need to be realigned.

Reply:
Thank you for your patience, I apologize for the beginner's clumsiness!

I'll fix it.

Reply:
... I tried to correct it... I'm really confused because of the lack of knowledge, because of the language barrier, and because of the fact that I can't in any other way equalize these comments to be on the same level. If I try it as you suggested, then they stick out here and there. If it suits you, I will do it, but in the original, they are arranged. It's not my fault that the code has changed, so some lines have become too long. If you look at these comments on my repository now, everything is fine.

Is it more important for us to strictly keep the positions of the tabs or a nice look of aligned comments as in the original?

Reply:
Maybe it's that what I'm seeing in the github preview isn't looking like it looks in a terminal?

Anyway, the important thing is that it's as accurate to the current code and indentation as possible.

Reply:
Here's what it looks like in my github preview. I do not use a correction terminal

![Git_set](https://user-images.githubusercontent.com/49764191/162573355-a6ec7430-c227-4d60-868b-7d1f622ab8d6.png)
.
So should I change that or leave it that way?